Salary App/Features/ValidationManager/ValidationManager.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ValidationManager.swift
+//  Salary App
+//
+//  Created by Tobi Adegoroye on 12/07/2023.
+//
+
+import Foundation
+
+enum FormError: Error {
```

Comment:
You should have a computed property for the error title that you want to show in the alert